src/calendar/header/index.tsx

```diff
@@ -3,21 +3,59 @@ import PropTypes from 'prop-types';
 import memoize from 'memoize-one';
 import XDate from 'xdate';
 
-import React, {Component, Fragment} from 'react';
-import {ActivityIndicator, Platform, View, Text, TouchableOpacity, Image} from 'react-native';
-
+import React, {Component, Fragment, ReactNode} from 'react';
+import {ActivityIndicator, Platform, View, Text, TouchableOpacity, Image, ColorValue, ViewStyle, AccessibilityActionEvent} from 'react-native';
+// @ts-expect-error
 import {shouldUpdate} from '../../component-updater';
+// @ts-expect-error
 import {weekDayNames} from '../../dateutils';
 import {
   CHANGE_MONTH_LEFT_ARROW,
   CHANGE_MONTH_RIGHT_ARROW,
   HEADER_DAY_NAMES,
   HEADER_LOADING_INDICATOR,
   HEADER_MONTH_NAME
+  // @ts-expect-error
 } from '../../testIDs';
 import styleConstructor from './style';
 
-class CalendarHeader extends Component {
+type Direction = 'left' | 'right';
+interface CalendarHeaderProps {
+  theme?: any;
```

Comment:
Why 'theme' is any and not object type?

Reply:
Changing it to `object` causes a ts error since it's trying to get `theme.indicatorColor` but it doesn't recognize it from the general object type.
I can change it to `_.get(theme, indicatorColor)` but I'm not sure it's better.. wdyt?

Reply:
Maybe it's better to define a `theme` type in a common service?